api/v0/observe/observe.proto

```diff
@@ -52,7 +52,30 @@ service ObserveService {
   rpc GetPosixSignalsStream(GetPosixSignalsStreamRequest) returns (stream GetPosixSignalsStreamResponse) {}
 }
 
-message GetPosixSignalsStreamRequest { 
+/// Request a stream of POSIX signals
+message GetPosixSignalsStreamRequest {
+  /// The scope of this request. If no scope is specified, a stream of all POXIS
+  /// signals on the host will be returned.
+  oneof workload {
+    Cell cell = 2;
+    Container container = 3;
+    VirtualMachine vm = 4;
+  }
+}
+
+message Host {
+}
+
+message Cell {
+  string cell_name = 1;
```

Comment:
Are we going to have more fields? Wondering why these are wrapped in messages?

Reply:
I've changed it now as per @dmah42's suggestion. These were mapped in messages b/c it was a `oneof`. Maybe I was thinking too much about how this would be mapped to Rust code though instead of what would be more idiomatic protobuf :)